src/main/java/labs/introtoprogramming/lab5/object/Parser/ParseObjFile.java

```diff
@@ -0,0 +1,507 @@
+package labs.introtoprogramming.lab5.object.Parser;
+
+import labs.introtoprogramming.lab5.object.Builder.BuilderInterface;
+import labs.introtoprogramming.lab5.object.Builder.LoadDataFromParserObject;
+
+import java.io.*;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.*;
+
+public class ParseObjFile {
```

Comment:
There is a lot of data. We need only vertices, normal and texture coordinates. It is too complicated to understand what going on here